recipe/license.patch

```diff
@@ -0,0 +1,12 @@
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -3,8 +3,7 @@
+ version = "8.2.2"
+ description = "Composable command line interface toolkit"
+ readme = "README.md"
+-license = "BSD-3-Clause"
+-license-files = ["LICENSE.txt"]
++license = { text = "BSD-3-Clause" }
```

Comment:
What was the error before applying the patch? Should we instead update flit_core if it supports a new format for project.license and project.license-files?